shared/dfinity/ic_types.py

```diff
@@ -192,3 +211,62 @@ class AbbrevProposal(TypedDict):
     summary: str
     title: str
     topic: ProposalTopic
+
+
+class AbbrevSubnetUpdateProposalPayload(TypedDict):
+    replica_version_id: str
+    subnet_id: str
+
+
+class AbbrevSubnetUpdateProposal(TypedDict):
+    """
+    {'proposal_id': 102768,
+    'payload': {'replica_version_id': 'b3b00ba59c366384e3e0cd53a69457e9053ec987',
+                'subnet_id': ('4zbus-z2bmt-ilreg-'
+                    'xakz4-6tyre-hsqj4-slb4g-zjwqo-snjcc-iqphi-3qe')},
+    'proposer': '80',
+    'status': 'EXECUTED',
+    'summary': 'Update subnet '
+                '4zbus-z2bmt-ilreg-xakz4-6tyre-hsqj4-slb4g-zjwqo-snjcc-iqphi-3qe '
+                'to replica version '
+                '[b3b00ba59c366384e3e0cd53a69457e9053ec987](https:##dashboard.internetcomputer.org/release/b3b00ba59c366384e3e0cd53a69457e9053ec987)\n',
+    'title': 'Update subnet 4zbus to replica version b3b00ba5',
+    'topic': 'TOPIC_IC_OS_VERSION_DEPLOYMENT'}
+    """
+
+    payload: AbbrevSubnetUpdateProposalPayload
+    proposal_id: int
+    proposer: int
+    status: ProposalStatus
+    summary: str
+    title: str
+    topic: ProposalTopic
+
+
+class AbbrevBoundaryNodesUpdateProposalPayload(TypedDict):
+    version: str
+    boundary_node_ids: str
+
+
+class AbbrevBoundaryNodesUpdateProposal(TypedDict):
+    """
+    {'proposal_id': 102768,
+    'payload': {'version': 'b3b00ba59c366384e3e0cd53a69457e9053ec987',
+                'node_ids': ['4zbus-z2bmt-ilreg-'
+                    'xakz4-6tyre-hsqj4-slb4g-zjwqo-snjcc-iqphi-3qe']},
```

Comment:
```suggestion
                'node_ids': ['4fssn-4vi43-2qufr-hlrfz-hfohd-jgrwc-7l7ok-uatwb-ukau7-lwmoz-tae']},
```
again: node id instead of subnet id